implants/lib/eldritch/src/file/temp_file_impl.rs

```diff
@@ -0,0 +1,46 @@
+use anyhow::Result;
+use std::fs;
+use std::path::PathBuf;
+use tempfile::NamedTempFile;
+
+pub fn temp_file(name: String) -> Result<String> {
```

Comment:
Can we make the name optional.
If not provided create a totally random name - could use the NamedTempFile function.